website/addons/wiki/static/pagedown-ace/Markdown.Editor.js

```diff
@@ -1704,9 +1721,13 @@ var Range = ace.require('ace/range').Range;
 
             buttons.redo = makeButton("wmd-redo-button", getStringAndKey("redo"), "-220px", null);
             buttons.redo.execute = function (manager) { inputBox.session.getUndoManager().redo(); };
+
             makeSpacer(4);
             makeCheckBox("wmd-autocom-toggle", "autocom", "-240px", "Autocomplete");
 
+            makeSpacer(5);
+            buttons.help = makeHelpButton("wmd-help-button", getStringAndKey("help"), "-240px");
+
             if (helpOptions) {

```

Comment:
`buttons.help` is also assigned on the last line of this conditional, and the lines between here and there do approximately what `makeHelpButton` does.